Header header
uint32 sequence
float64 scale
float64[] mazerot2d
uint32[] centroid2d
geometry_msgs/Pose maze_origin